docker/Makefile

```diff
@@ -1,60 +1,72 @@
 dc=docker-compose
 d=docker
-dcp=docker-compose -f docker-compose-production.yml
+dcb=$(dc) -f build/docker-compose.yml
+dcp=$(dc) -f docker-compose-production.yml
+
+# @todo #269 Create docs for build system.
+
+.PHONY: migrate create-env build-static watch-static build test backup restore
+.PHONY: lint dev build-production-static deploy
 
 migrate:
-	$(dc) exec se-python python manage.py migrate
-
-# for local env
-static:
-	$(dc) pull se-nodejs
-	$(dc) run --rm se-nodejs bash -c "\
-		npm install -g gulp-cli && \
-		# we use `--no-optional` because some optional npm dependencies fail on install
-		npm install --no-optional && \
-		gulp build"
-
-# full version of this recipe will be developed in #248
-# for local env
+	$(dc) run --rm app python manage.py migrate
+
+create-env:
+	$(shell ./create-env.sh)
+
+build-static:
+	$(dc) run --rm nodejs gulp build
+
+watch-static:
+	$(dc) run --rm nodejs gulp watch
+
 build:
-	$(dc) build --no-cache se-python  # fetch <DEPS>/refarm-site/front/ dir
-	$(dc) pull
-	$(MAKE) static
+	# Build nodejs image and static files
+	$(dcb) build --no-cache nodejs
+	$(MAKE) build-static
+	# Build python image with sources and static files
+	$(dcb) build --no-cache python-dev
 
-test: static
-	$(dc) up -d se-python se-selenium
-	$(dc) exec se-python python manage.py test -v 3 --liveserver=se-python:8020-8030 --parallel
+test: build-static
+	$(dc) up -d app selenium
+	$(dc) exec app python manage.py test -v 3 --liveserver=app:8020-8030 --parallel
 	$(dc) stop
 
+lint:
+	$(dc) up -d app
+	$(dc) run --rm lint
+	$(dc) stop
+
+deploy-dev:
```

Comment:
let's add to this rule comment like this:
```
# depends on `build-static`. Don't set this dependency explicitly for caching purposes
```

Reply:
@duker33 

`se-nodejs:dev` already has built static files, so will be enough to pull it.